tests/test_special_orthogonal_group.py

```diff
@@ -717,7 +756,12 @@ def test_inverse_vectorization(self):
             n_samples = self.n_samples
             points = group.random_uniform(n_samples=n_samples)
             result = group.inverse(points)
-            self.assertTrue(result.shape == (n_samples, group.dimension))
+
+            if n == 3:
+                self.assertTrue(result.shape == (n_samples, group.dimension))
```

Comment:
why is this a special case? Shouldn't it also be n, n?

Reply:
Because the default point_type for n==3 is 'vector', and the default point_type for n!=3 is 'matrix'.